DealiDesignSystemSampleApp/DealiDesignSystemSampleApp/ViewControllers/MainViewController.swift

```diff
@@ -12,20 +12,30 @@ import DealiDesignKit
 
 final class MainViewController: UIViewController {
 
-    let mainView = MainView()
     
     override func loadView() {
-        self.view = mainView
+        self.view = .init()
+        
+        self.view.backgroundColor = DealiColor.primary01
```

Comment:
백그라운드 색상이랑 버튼 색상이 구분이 안 가서 흰색으로 바꿔도 되나요?ㅎㅎ
```suggestion
        self.view.backgroundColor = DealiColor.primary04
```

Reply:
변경했음